parent/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you do not need a specific parent module (you already have one, ./pom.xml)

Reply:
@artemmoroz done, please review